BDArmory/Utils/BDAMath.cs

```diff
@@ -72,6 +72,8 @@ private static float SqrtARM(float value)
 
         public static float SolveTime(float distance, float acceleration, float vel = 0)
         {
+            if (acceleration == 0f) return float.MaxValue;
```

Comment:
This is only the case if `vel` is also 0. Otherwise, it should return `x = -c / b`.